src/Middleware/RateLimiting/src/RateLimiterOptionsExtensions.cs

```diff
@@ -34,12 +41,18 @@ public static RateLimiterOptions AddTokenBucketLimiter(this RateLimiterOptions o
     }
 
     /// <summary>
-    /// Adds a new <see cref="FixedWindowRateLimiter"/> with the given <see cref="FixedWindowRateLimiterOptions"/> to the <see cref="RateLimiterOptions"/>.
+    /// Registers a new <see cref="FixedWindowRateLimiter"/> with the given <see cref="FixedWindowRateLimiterOptions"/> to the to the application.
```

Comment:
I would have like to see something like

> Adds a limit to the number of requests that can be executed in a specific time window.

Then for the sliding:

> Adds a limit to the number of requests that can be executed in a sliding time window.

This kind of help message. We should check with others if this is not the kind of message we want in API docs.